virttest/remote_commander/remote_runner.py

```diff
@@ -703,6 +711,54 @@ def exit(self):
         self._exit = True
         return "bye"
 
+    def run_test(self, test_path):
+        """Call run() function at external python module.
+
+        Parameters
+        ----------
+        test_path : str
+            Path to python file. Call run() function at this module.
+
+        Returns
+        -------
+            module.run().return
+        """
+        assert os.path.isfile(test_path)
+        module = imp.load_source('ext_test', test_path)
```

Comment:
Avocado-vt uses it for backward compatibility, it was not design decision. Anyway if I understand the response correctly you have existing tests which can't be executed directly and you can't modify them. I can understand that.